presto-native-execution/presto_cpp/presto_protocol/connector/clp/ClpConnectorProtocol.cpp

```
@@ -0,0 +1,163 @@
+#include "presto_cpp/presto_protocol/connector/clp/ClpConnectorProtocol.h"
+
+#include <folly/lang/Bits.h>
+#include <sstream>
+
+namespace facebook::presto::protocol::clp {
+
+namespace {
+
+// Java's DataOutputStream.writeUTF: 2-byte big-endian length + UTF-8 bytes
+std::string readUTF(std::istringstream& in) {
+  uint16_t length;
```

Clang (14.0.6)

[warning] variable 'length' is not initialized

(cppcoreguidelines-init-variables)